tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -762,11 +766,20 @@ def number_peaks(x, n):
     :return: the value of this feature
```

Comment:
Yes, this is probably true. But as stated in the conversation of this PR, I have already reduced this function to a quite small and reasonable time. We should concentrate on the last 4 runtime issues (augmented_dicker_fuller 14 %, mean_abs_change_qunatiles 25 %, autocorrelation 14 %, quantile 6 %) if we want to make the extraction even faster.